worker/app/tasks.py

```
@@ -68,6 +47,80 @@ def _is_cancelled(task_id: str) -> bool:
+        "video_codec": kwargs.get("video_codec"),
+        "audio_codec": kwargs.get("audio_codec"),
+        "audio_bitrate_kbps": kwargs.get("audio_bitrate_kbps"),
+        "preset": kwargs.get("preset"),
+        "max_height": kwargs.get("max_height"),
+        "max_width": kwargs.get("max_width"),
+        "max_output_fps": kwargs.get("max_output_fps"),
+    }
+    
+    _publish(self.request.id, {"type": "log", "message": f"Offloading encoding task to native daemon at {daemon_url}..."})
+    
+    req = urllib.request.Request(f"{daemon_url}/encode", data=json.dumps(payload).encode('utf-8'), headers={'Content-Type': 'application/json'}, method='POST')
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)